src/soca/State/State.cc

```diff
@@ -67,6 +68,20 @@ namespace soca {
     Log::trace() << "State::State created by interpolation." << std::endl;
   }
   // -----------------------------------------------------------------------------
+  State::State(const oops::Variables & vars, const State & other) : State(other)
+  {
+    // TODO(Travis, maybe) The variable change needs to go here
+    //  (U/V rotate, etc), but since we don't really use it right now, this 
+    //  can wait until after the variable changes get cleaned up, (and 
```

Comment:
```suggestion
    //  (U/V rotate, etc), but since we don't really use it right now, this
    //  can wait until after the variable changes get cleaned up, (and
```
Removing end-of-line whitespace that  is causing the coding norms test to fail.